lib-multisrc/mayotune/src/eu/kanade/tachiyomi/multisrc/mayotune/MayoTune.kt

```diff
@@ -0,0 +1,162 @@
+import org.jsoup.nodes.Element
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.TimeZone
+import kotlin.getValue
+
+abstract class MayoTune(
+    override val name: String,
+    override val baseUrl: String,
+    override val lang: String,
+) : ParsedHttpSource() {
+    open val sourceList = listOf(SManga.create())
+    private val json: Json by injectLazy()
+
+    // Popular
+    override fun fetchPopularManga(page: Int): Observable<MangasPage> {
+        return Observable.just(MangasPage(sourceList, false))
+    }
+
+    override fun popularMangaRequest(page: Int): Request = throw UnsupportedOperationException()
+    override fun popularMangaNextPageSelector(): String? = throw UnsupportedOperationException()
+    override fun popularMangaSelector(): String = throw UnsupportedOperationException()
+    override fun popularMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    // Latest
+    override val supportsLatest: Boolean = true
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> {
+        return Observable.just(MangasPage(sourceList, false))
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request = throw UnsupportedOperationException()
+    override fun latestUpdatesNextPageSelector(): String? = throw UnsupportedOperationException()
+    override fun latestUpdatesSelector(): String = throw UnsupportedOperationException()
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        throw UnsupportedOperationException()
+
+    // Search
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        val mangas = mutableListOf<SManga>()
+        sourceList.map {
+            if (it.title.lowercase().contains(query.lowercase())) {
+                mangas.add(it)
+            }
+        }
+        return Observable.just(MangasPage(mangas, false))
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList) =
+        throw UnsupportedOperationException()
+
+    override fun searchMangaNextPageSelector() = throw UnsupportedOperationException()
+    override fun searchMangaSelector() = throw UnsupportedOperationException()
+    override fun searchMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    // Get Override
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return GET(manga.url, headers)
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return GET(manga.url + "api/chapters", headers)
+    }
+
+    override fun pageListRequest(chapter: SChapter): Request {
+        return GET(chapter.url, headers)
+    }
+
+    // Details
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        val statusTxt = document.select("div.text-center").find {
```

Comment:
Don't need to save 1 character here.
```suggestion
        val statusText = document.select("div.text-center").find {
```